### PR TITLE
Add info about nightly builds

### PR DESCRIPTION
**Pull request by enoch85**
Update README.md

Fixes https://github.com/owncloud/mail/issues/1061

cc @jancborchardt @ChristophWurst @Gomez

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -83,6 +83,16 @@ php composer.phar install
 ```
 Debian/Ubuntu people, just execute `install_ubuntu.sh`
 
+## Nightly Builds
+
+Instead of setting everything up manually, you can just [download the nighly builds](https://en0ch.se/index.php/s/3v5J3XXeQCCp73p) instead. These builds are updated every 12 hour - 00:00 and 12:00, and is pre-configured with all the needed dependencies.
```
Comment (ChristophWurst) [inline README.md]: * every 12 hour*s*
* I think it should be 'and _are_ pre-configured' instead of 'and _is_ pre-configured'

File: README.md [hunk 2]
```diff
@@ -83,6 +83,16 @@ php composer.phar install
...
+
+1. Download
+2. Exctract the zip archive to /owncloud/apps/mail
```
Comment (ChristophWurst) [inline README.md]: the archive should contain the 'mail' directory, therefore users extract it to 'path-to-owncloud/apps'

**Comment by ChristophWurst** [comment]
Beside those notes, :+1: Thanks a lot for providing your server to share nightlies :-)

**Comment by enoch85** [comment]
@ChristophWurst Made the changes. Should we also remove the Dev info and only have the Nighly builds to stop confusion?

**Comment by ChristophWurst** [comment]
> @ChristophWurst Made the changes. Should we also remove the Dev info and only have the Nighly builds to stop confusion?

Thanks! No, those dev infos have to stay there. Nighlies are only meant for people who want to try the current dev version, they are not intended for development.

**Comment by jakobsack** [comment]
looks great :+1:

**Comment by enoch85** [comment]
Ok, I´ll merge this when we have working nightlies. Still some issue left: https://github.com/owncloud/mail/pull/1065#issuecomment-139352562

**Comment by PoPoutdoor** [comment]
@enoch85 The point is user knows their local timezone but not always knows time difference between CEST and local time.

For example, my timezone is UTC+0800, I have to look up CEST = UTC+0200 before I know when will the next nightly build ready for download at local time. Now I know the nightly buid will be ready at my time @0600 and @1800

So, do a one time change can save many user's time.

File: README.md [hunk 3]
```diff
@@ -83,6 +83,16 @@ php composer.phar install
 ```
 Debian/Ubuntu people, just execute `install_ubuntu.sh`
 
+## Nightly Builds
```
Comment (jancborchardt) [inline README.md]: This needs to be a h3 (three ###), otherwise the »Resetting the app« down below is a sub-point of this here which doesn’t make sense.

Comment (jancborchardt) [inline README.md]: Also, capitalization should be »Nightly builds« (only capitalize first word)

**Comment by jancborchardt** [comment]
See comments. :) Also @enoch85 after it’s done please squash all the commits into one for cleaner history.

**Comment by enoch85** [comment]
@jancborchardt How do I squash the commits?

**Comment by jancborchardt** [comment]
@enoch85 very simple, best check https://ariejan.net/2011/07/05/git-squash-your-latests-commits-into-one/ for a nice tutorial. :)

**Comment by enoch85** [comment]
@PoPoutdoor Could you please try the latest build found in `/build/artifacts/appstore`?

**Comment by enoch85** [comment]
@jancborchardt So basically this from shell:

```
git rebase -i HEAD~5 https://github.com/owncloud/mail/pull/1066/commits

pick COMMITNUMBER made changes based on ChristophWurst comments
squash COMMITNUMBER add AM/PM
squash COMMITNUMBER Add timezone info
squash COMMITNUMBER Add UTC + edit according to comments
```

Don't want to mess this up, that´s why I'm asking. :)

**Comment by Henni** [comment]
@enoch85 `git rebase -i master` should do the trick
then do it like you described it by setting all commits except the first to `squash`

**Comment by enoch85** [comment]
@Henni `git rebase -i master https://github.com/owncloud/mail/pull/1066/` gives me: `fatal: Not a git repository (or any of the parent directories): .git`

Better to do a new PR instead with just one commit @jancborchardt ?

**Comment by Henni** [comment]
@enoch85 you shouldn't need the url.
Checkout the branch you want to squash and then execute `git rebase -i master`.

**Comment by enoch85** [comment]
@Henni The thing is, I use git from my servers shell. Where should I write the commands?

**Comment by enoch85** [comment]
Fix is here: https://github.com/owncloud/mail/pull/1070 I give up on the squashing part. :)

**Pull request closed without merging**